tests/apps/configurationapp/app.go

```diff
@@ -140,7 +140,7 @@ func getRedisValuesFromItems(items map[string]*Item) []interface{} {
 }
 
 func getHTTP(keys []string) (string, error) {
-	url := "http://" + daprHost + ":" + daprHTTPPort + "/v1.0-alpha1/configuration/" + configStoreName + buildQueryParams(keys)
```

Comment:
add constants for `http://` and also `/v1.0/configuration/`

Reply:
Done